portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,36 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private List<String> quotes;
+
+  @Override
+  public void init() {
+    quotes = new ArrayList<>();
+    quotes.add("Hope you are having a good one!");
+    quotes.add("How are you :)");
+    quotes.add("It is fantastic to see you!");
+    quotes.add("Thanks for vising my portfolio!");
+  }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Kathy!</h1>");
+    // Convert quotes object to JSON format.
+    Gson gson = new Gson();
+    String json = gson.toJson(quotes);
```

Comment:
I wonder if it'd be more efficient to just store the json instead of the list? Unless you plan to alter the list further in the future.

Reply:
I think you have a good point. I didn't think of this. I believe in this circumstance, it's more efficient to just store the json.
In later steps of the walkthrough, the list will hold a changing number of quotes (as user comments). So in the future, a list might be easier to manipulate. This is probably why the instructions are asking us to create a list and convert to json.

Reply:
Should I make a modification here?

Reply:
Ah gotcha, feel free to keep it as is then for now if that's coming up

Reply:
Ok! Thank you